pkg/parser/parseable/default.go

```diff
@@ -43,3 +52,13 @@ func (parser *DefaultParser) Collectible() bool {
 func (parser *DefaultParser) SetCollectible(value bool) {
 	parser.collectible = value
 }
+
+func (parser *DefaultParser) NormalField(name string) *Field {
```

Comment:
Should it be called something like `FindFieldByName`?

Reply:
Renamed in https://github.com/cirruslabs/cirrus-cli/pull/494/commits/72f68def2656d04a50fb66a969513fe6fc95cbdc.